test/genesis/genesis.go

```diff
@@ -0,0 +1,267 @@
+package genesis
```

Comment:
Wouldn't this be better placed under `util`

Reply:
https://github.com/celestiaorg/celestia-app/pull/2455/commits/c0ab03282a00aef2233422f74624bcb8e7187e93